scripts/build-version.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+import sys
+
+def to_semver(gitVersion):
```

Comment:
minor: `git_version` in Python is more idiomatic.

You can also throw on a `: string` if you want. It doesn't do anything but helps aid in clarity.